rac-gcp-deploy/orb.yml

```diff
@@ -226,19 +254,27 @@ jobs:
         description: "Path to persist the workspace"
         type: string
         default: ~/project
+      cache-key:
+        type: string
+        default: *cache-key
       cache-suffix:
         description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
         default: ""
         type: string
+      sbt-extra-options:
+        description: "Extra options being passed to sbt, like '--mem 2048'"
```

Comment:
Is it a mistake that there are two `-` here?

Reply:
yes it is